SEO: reword introduction paragraph into descriptions - Part 1 (#408)
front-commerce/developers.front-commerce.com: source/docs/essentials/create-a-business-component.md

---
id: create-a-business-component
title: Create a Business Component

> description: In this section, we will build a Business Component. The core concept is the same than creating an UI component.

Reviewer: ```suggestion
description: In this section, we will build a Business Component. The core concept is the same as creating a UI component.
```
